scripts/backup.sh

```diff
@@ -179,23 +183,59 @@ if [ "${SOURCE_MODE}" = "storage" ]; then
     ID="${ID_FILE%.*}"
     cd
 
-    # Set the backup path
-    BACKUP_PATH="${STORAGE_PATH}"/"${ID}"
+    # Set SOURCE_PATH
+    SOURCE_PATH="${SOURCE_MOUNT_POINT}"
+
+    # Set BACKUP_PATH
+    BACKUP_PATH="${STORAGE_PATH}/${ID}"
 
-    # SOURCE_IDENTIFIER
+    # Set SOURCE_IDENTIFIER
     SOURCE_IDENTIFIER="Source ID: ${ID}"
 
-# elif [ "${SOURCE_MODE}" = "NEW_SOURCE_DEFINITION" ];
-# then
+elif [ "${SOURCE_MODE}" = "ios" ]; then
+    if [ $DISP = true ]; then
+        lcd_message "Ready" "Connect" "iOS device"
+    fi
+
+    # Mount iOS device
+    ifuse $MOUNT_IOS_DIR -o allow_other
+    SOURCE_PATH="${MOUNT_IOS_DIR}/DCIM"
+
+    # Exit with a message if iOS device is not mounted
+    if [ -z "$(ls -A $MOUNT_IOS_DIR)" ]; then
+        if [ $DISP = true ]; then
+            lcd_message "No iOS device" "Try again"
+            exit 1
+        fi
+    fi
+
+   # Create  a .id random identifier file if doesn't exist
+    cd "${MOUNT_IOS_DIR}"
+    if [ ! -f *.id ]; then
+        random=$(echo $RANDOM)
+        touch $(date -d "today" +"%Y%m%d%H%M")-$random.id
+    fi
+    ID_FILE=$(ls -t *.id | head -n1)
+    ID="${ID_FILE%.*}"
+    cd
+
+    # Set BACKUP_PATH
+    BACKUP_PATH="${STORAGE_PATH}/iOS"
```

Comment:
`BACKUP_PATH="${STORAGE_PATH}/iOS"` should probably be changed to `BACKUP_PATH="${STORAGE_PATH}/iOS BACKUP"`, because there is already the `/home/pi/iOS` directory on the internal storage.